doc/plutus/tutorials/plutus-tx.rst

```diff
@@ -1,266 +0,0 @@
-
-The key technique that we use to implement Plutus Tx is called *staged
-metaprogramming*, which means that the main Haskell program generates *another* program
-(in this case, the Plutus Core program that will run on the blockchain). Plutus Tx is the
-mechanism we use to write those programs, but since Plutus Tx is just part of the main
-Haskell program, we can share types and definitions between the two.
-
-.. _template_haskell_preliminaries:
-
-Template Haskell preliminaries
-------------------------------
-
-Plutus Tx uses Haskell's metaprogramming support, Template Haskell, for two main reasons:
-
--  Template Haskell enables us to work at compile time, which is when we do Plutus Tx compilation.
--  It allows us to wire up the machinery that invokes the Plutus Tx compiler.
-
-Template Haskell is very versatile, but we only use a few features.
-
-Template Haskell begins with *quotes*. A Template Haskell quote is a
-Haskell expression ``e`` inside special brackets ``[|| e ||]``. It has
-type ``Q (TExp a)`` where ``e`` has type ``a``. ``TExp a`` is a
-*representation* an expression of type ``a``, i.e. the syntax of the
-actual Haskell expression that was quoted. The quote lives in the type
-``Q`` of quotes, which isn’t very interesting for us.
-
-.. note::
-
-   There is also an abbreviation ``TExpQ a`` for ``Q (TExp a)``, which
-   avoids some parentheses.
-
-You can *splice* a quote into your program using the ``$$`` operator.
-This inserts the syntax represented by the quote into the program at the
-point where the splice is written.
-
-Simply put, a quote allows us to talk about Haskell programs as *values*.
-
-The Plutus Tx compiler compiles Haskell *expressions* (not values!), so
-naturally it takes a quote (representing an expression) as an argument.
-The result is a new quote, this time for a Haskell program that
-represents the *compiled* program. In Haskell, the type of :hsobj:`PlutusTx.TH.compile`
-is ``TExpQ a → TExpQ (CompiledCode a)``. This is just
-what we already said:
-
--  ``TExpQ a`` is a quoted representing a program of type ``a``.
-
--  ``TExpQ (CompiledCode a)`` is quote representing a
-   compiled Plutus Core program.
-
-.. note::
-
-   :hsobj:`PlutusTx.CompiledCode` also has a type parameter ``a``, which
-   corresponds to the type of the original expression. This lets us
-   "remember" the type of the original Haskell program we compiled.
-
-Since :hsobj:`PlutusTx.TH.compile` produces a quote, to use the result we need to splice
-it back into our program. The Plutus Tx compiler runs when compiling the main program,
-and the compiled program will be inserted into the main program.
-
-This is all you need to know about the Template Haskell! We often use the same simple pattern: make a quote,
-immediately call :hsobj:`PlutusTx.TH.compile`, and then splice the result back in.
-
-.. _writing_basic_plutustx_programs:
-
-Writing basic PlutusTx programs
--------------------------------
-
-.. literalinclude:: BasicPlutusTx.hs
```

Comment:
`BasicPlutusTx.hs` should be removed too.

Reply:
Removed

Reply:
I've removed `plutus-tx.rst` so that the page is no longer in read the docs. But I've left the file `BasicPlutusTx.hs` because I don't think it causes any problems by being there.

Reply:
Again, should be removed if it's not used.